src/write_outputs/write_allamcyclelox.jl

```diff
@@ -0,0 +1,178 @@
+
+@doc raw"""
+	write_capacity(path::AbstractString, inputs::Dict, setup::Dict, EP::Model))
+
+Function for writing the diferent capacities for the FLECCS technologies (starting capacities or, existing capacities, retired capacities, and new-built capacities).
+"""
+function write_allam_capacity(path::AbstractString, inputs::Dict, setup::Dict, EP::Model)
+	# Capacity decisions
+	gen = inputs["RESOURCES"]
+	ALLAM_CYCLE_LOX = inputs["ALLAM_CYCLE_LOX"] 
+    COMMIT_Allam = setup["UCommit"] > 1 ? ALLAM_CYCLE_LOX : Int[]
+	MultiStage = setup["MultiStage"]
+
+    # Allam cycle components
+    # by default, i = 1 -> sCO2Turbine; i = 2 -> ASU; i = 3 -> LOX
+    sco2turbine, asu, lox = 1, 2, 3
+    # get component-wise data
+    allam_dict = inputs["allam_dict"]
+
+	capAllam_sco2turbine = zeros(size(inputs["RESOURCE_NAMES"]))
+	capAllam_asu = zeros(size(inputs["RESOURCE_NAMES"]))
+	capAllam_lox = zeros(size(inputs["RESOURCE_NAMES"]))
```

Comment:
Consider using `G = inputs["G"]` and then `zeros(G)` instead of `size(inputs["RESOURCE_NAMES"])`.